Tests/DependencyInjection/Compiler/AddDependencyCallsCompilerPassTest.php

```diff
@@ -189,6 +189,32 @@ function ($element) {
     /**
      * @covers Sonata\AdminBundle\DependencyInjection\Compiler\AddDependencyCallsCompilerPass::process
```

Comment:
i think this is correct, as it should JUST covers this method, not anything which is used in this test